src/platform/forms-system/src/js/utilities/data/reduceErrors.js

```diff
@@ -1,8 +1,90 @@
+ * @param {string} number - Number portion of the regexp that matches the number
+ * @returns {string} "{converted word number} {word}"
+ */
+export const replaceNumberWithWord = (_, word, number) => {
+  const num = parseInt(number, 10);
+  return `${
+    isNaN(num) || !isFinite(num) ? number : numberToWords(num + 1)
+  } ${word}`;
+};
 
-// Keys to ignore within the pageList objects & pageList schema
+/**
+ * @typedef FormConfig~reviewErrorMessage
+ * @type {string|function}
+ * @param {number} index - provided to the function if the schema property is an
+ *  array. This index indicates the index of the entry with an error
+ * @returns {string} predefined error message
+ */
+/**
+ * @typedef FormConfig~reviewErrors - Cross-reference of required schema names
+ *  with the error message displayed inside of the alert error link
+ * @type {Object}
+ * @property {Object.<string>} - required schema name (key)
+ * @property {FormConfig~reviewErrorMessage} - Error message string or function
+ *  that returns a string
+ */
+/**
+ * Get error message contained in FormConfig~reviewErrors
+ * @param {string|function} message - property name with an error
+ * @param {string} index - index of error (as a string, so it is concerted)
+ * @returns {string} Predefined error message
+ */
+export const getErrorMessage = (message, index = '') =>
+  typeof message === 'function' ? message(Number(index)) : message || '';
+
+// Regexp & replace string/function to reformat hard-coded error messages
+const messageFormatting = [
+  // Error messages include "requires property" or "instance."
+  { regex: /(requires property|instance\.?)\s*/g, replace: '' },
+  // Don't show the "view:" or "ui:" prefix
+  { regex: /(view:|ui:|")/g, replace: '' },
+  // Convert numbers into words
+  { regex: /(\w+)\[(\d+)\]/, replace: replaceNumberWithWord },
+  // Separate numbers (e.g. "address1" -> "address 1")
+  { regex: /([a-z])(\d)/g, replace: '$1 $2 ' },
+  // "zip" code replaced with "postal" code in content, but not property names
+  { regex: /zip\s(code)?/i, replace: 'postal code' },
+  // Make abbreviations upper case
+  { regex: /\b(va|pow)\b/g, replace: str => ` ${str.toUpperCase()} ` },
+  // "enum values:" may be followed by a very long list (e.g. all countries)
+  { regex: /(enum\svalues:.+)$/g, replace: 'the available values' },
+];
+
+/**
+ * Make hard-coded jsonschema validation error messages _more_ readable, e.g.
+ * - Changes`instance.news does not meet minimum length of 1` to
+ *   `News does not meet minimum length of 1`
+ * - Changes `instance.address requires property "city"` to `Address city`
+ * - Array type properties need to get special treatment,
+ *   `"instance.newDisabilities[0] requires property "cause"` is modified into
+ *   `First new disabilities cause`.
+ * These formatting methods use the schema property name, which isn't ideal.
+ * That's why this is used as a fallback for the predefined `reviewErrors`
```

Comment:
I wanted to emphasize that these messages are not ideal, and only used as a fallback